core/src/main/java/com/gentics/mesh/core/actions/impl/GroupDAOActionsImpl.java

```diff
@@ -55,9 +55,7 @@ public Page<? extends HibGroup> loadAll(DAOActionContext ctx, PagingParameters p
 
 	@Override
 	public Page<? extends HibGroup> loadAll(DAOActionContext ctx, PagingParameters pagingInfo, Predicate<HibGroup> extraFilter) {
-		return ctx.tx().groupDao().findAll(ctx.ac(), pagingInfo, group -> {
```

Comment:
OrientDB had here problems on resolving the direct extraFilter application. No OrientDB, no problem.